hack/install-kustomize.sh

```diff
@@ -25,6 +26,6 @@ fi
 
 echo "Installing ${tool}:${required_version} in ${target_dir}"
 mkdir -p "${target_dir}"
-curl --silent --show-error --location --output "${target_path}" "${source_url}"
-tar -xzf "${target_path}" -C "${target_dir}"
+curl --silent --show-error --location --output "${target_archive}" "${source_url}"
+tar -xzf "${target_archive}" -C "${target_dir}"
```

Comment:
Shouldn't we delete the downloaded tar? (can be done later).

Reply:
Updated.